content/participants/ttjc@poojakelkar/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Pooja Kelker
+type: "participant"
+name: Pooja Kelkar"
+
+github-link: "https://github.com/poojakelkar"
+linkedin-link: "https://linkedin.com/in/poojakelkar"
+looking-for: "job"
+description: "CSS/HTML | NodeJS | JQuery | MongoDB"
+---
+
+Hi! I am Pooja Kelkar, let me walk you through my intro, projects and blogs.
+
+### introduction
+
+Currently, a student pursuing B.Tech. Information Technology at SVKM's IOT Dhule,
+Passionate about participating in competitions like Hackathons and organizing events.Keen interest in research on new technologies.
+I like contributing to the community and share my knowledge.
+
+## why are you taking this challenge?
+
+I need a job in December 2020.
+I want to learn.
+I want get mentorship from industry expert and develop skills
```

Comment:
I want **to** get mentorship ...

Reply:
i have made the changes please accept my PR